src/covid_model_seiir_pipeline/forecasting/model.py

```diff
@@ -189,3 +188,59 @@ def beta_shift(beta_past: pd.DataFrame,
     scale_params['history_days'] = avg_over
 
     return beta_final, scale_params
+
```

Comment:
Pulled in from https://github.com/ihmeuw/covid-model-seiir-pipeline/pull/68

No changes otherwise